nameko/rpc.py

```diff
@@ -350,10 +351,16 @@ def __call__(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply.result()
 
-    def async(self, *args, **kwargs):
+    def call_async(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply
 
+    def async(self, *args, **kwargs):  # pragma: no cover
```

Comment:
can we not actually test this instead as matt [suggests](https://github.com/onefinestay/nameko/pull/318#issuecomment-216233850)?